src/components/pages/address-overview/address-overview.tsx

```diff
@@ -130,7 +185,21 @@ const AddressOverview: React.FC<{ address?: Address }> = ({ address }) => {
             variant={ButtonVariant.PRIMARY}
             onPress={() => {
               fetchJson(`/api/profile/addresses/${address?.id}`, "DELETE", {})
-                .then(() => router && router.push("/account/profile"))
+                .then(async () => {
+                  router && router.push("/account/profile");
```

Comment:
It doesn't change anything to have the `router.push()` before the rest of the statements ?

Reply:
You're right, it works but it's not clean and it make some things happen after the page changed.

I'm changing that 👍

Reply:
Alright, that was my point, I wanted to know first (for curiosity sake) if it worked smoothly like that. It's good to know it. 🙂 
But indeed, I find it cleaner too to have this piece as the last statement 👌